packages/react-components/src/components/ChatMessage/MyMessageComponents/ChatMyMessageComponent.tsx

```diff
@@ -158,7 +166,11 @@ export const ChatMyMessageComponent = (props: ChatMyMessageComponentProps): JSX.
           message.messageId,
           text,
           /* @conditional-compile-remove(file-sharing-acs) */
-          { attachments: attachments }
+          /* @conditional-compile-remove(rich-text-editor-image-upload) */
+          {
+            /* @conditional-compile-remove(file-sharing-acs) */ attachments: attachments,
+            type: props.richTextEditorOptions ? 'html' : 'text'
```

Comment:
Do we need to include this information (as the edited message already has a type and can't change it)?